client/src/App.js

```diff
@@ -103,8 +104,12 @@ function App () {
    * component is passed
    */
   const playAudio = (key) => {
-    const audioEl = document
-      .querySelector(`#sound_${GameStateConstants.GAME_SOUNDS[key]}`);
+    setCurrentAudio(GameStateConstants.GAME_SOUNDS[key]);
```

Comment:
Can we,
- set the current audio state variable alone in this App.js ? 
- Pass that state variable as prop to the Audio component
- Move this document query and play logic inside that Audio component, which would be triggered using `useEffect` inside the Audio component? 
This way we can move the play logic as well within the Audio component and all the App.js does is change the state variable.